datafusion/core/tests/repartition_exec_blocks.rs

```diff
@@ -0,0 +1,60 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
what do you think about naming this test something more general like `repartiton.rs`?

Reply:
done